pages/resources/application.mdx

```diff
@@ -4,153 +4,587 @@
+| bot_require_code_grant? ^3^ **(deprecated)** | boolean                                                                 | Whether the application's bot will only be added upon completion of a full OAuth2 token exchange                                                                                     |
+| approximate_guild_count?                     | integer                                                                 | Approximate count of guilds the application's bot is in                                                                                                                              |
+| terms_of_service_url?                        | string                                                                  | The URL to the application's terms of service                                                                                                                                        |
+| privacy_policy_url?                          | string                                                                  | The URL to the application's privacy policy                                                                                                                                          |
+| role_connections_verification_url            | ?string                                                                 | The role connection verification entry point of the integration; when configured, this will render the application as a verification method in guild role verification configuration |
+| interactions_endpoint_url                    | string                                                                  | The URL of the application's [interactions endpoint](/interactions/receiving-and-responding#receiving-an-interaction)                                                                |
+| interactions_version                         | integer                                                                 | The [version of the application's interactions endpoint implementation](#application-interatcions-version)                                                                           |
+| interactions_event_types ^4^                 | array[string]                                                           | The enabled [Gateway events](/topics/gateway-events) to send to the interaction endpoint                                                                                             |
+| tags?                                        | array[string]                                                           | Tags describing the content and functionality of the application (max 20 characters, max 5)                                                                                          |
```

Comment:
```suggestion
| tags?                                        | array[string]                                                           | Tags describing the content and functionality of the application, processed to categorise the app in the App Directory (max 20 characters, max 5)                                                                                          |
```

Reply:
I don't think this is only for app directory